### DISCUSSION TITLE
CrashLoopBackOff in prometheus-config-reloader

### DISCUSSION THREAD

**Discussion by RossWilliamson** [carlosedp/cluster-monitoring#139] [Q&A] [answered]
So I'm having an issue with the prometheius-k8s-0 pod (Note I'm very new at this so bear with me).  It's stuck in a CrashLoopBackOff error. I did a:

kubectl describe pod -n monitoring prometheus-k8s-0

and I think the error is with prometheus-config-reloader where the error is  

> Message:      failed to create containerd task: failed to create shim: OCI runtime create failed: container_linux.go:380: starting container process caused: exec: "/usr/bin/configmap-reload": stat /usr/bin/configmap-reload: no such file or directory: unknown"

Now I'm at a loss.  I'm using k3s and enabled that in the vars.jsonnet (See diff below).  Note I'm running on an odroid xu4 not a RPI with a master at 1.100 and two nodes at 1.101 and 1.102

```
--- a/vars.jsonnet
+++ b/vars.jsonnet
@@ -12,7 +12,7 @@
     },
     {
       name: 'armExporter',
-      enabled: false,
+      enabled: true,
       file: import 'modules/arm_exporter.jsonnet',
     },
     {
@@ -48,12 +48,12 @@
   ],
 
   k3s: {
-    enabled: false,
-    master_ip: ['192.168.1.15'],
+    enabled: true,
+    master_ip: ['192.168.1.100'],
   },
 
   // Domain suffix for the ingresses
-  suffixDomain: '192.168.1.15.nip.io',
+  suffixDomain: '192.168.1.102.nip.io',
```

**Comment by carlosedp** [comment]
Might be something related to new versions of the image... it would be good if someone else could confirm this.
I'm not running any cluster right now.

**Comment by badnetmask** [comment, marked as answer]
I cloned the repo 3 days ago, and everything has been working fine.

**Comment by carlosedp** [comment]
@RossWilliamson check your cluster config and monitor stack deployment.
